src/Microsoft.ML.AutoML.Interactive/AutoMLMonitorKernelExtension.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.AspNetCore.Html;
+using Microsoft.Data.Analysis;
+using Microsoft.DotNet.Interactive;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.Formatting;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Plotly.NET.CSharp;
+using static Microsoft.DotNet.Interactive.Formatting.PocketViewTags;
+
+
+namespace Microsoft.ML.AutoML
+{
+    public class AutoMLMonitorKernelExtension : IKernelExtension
+    {
+        public async Task OnLoadAsync(Kernel kernel)
+        {
+            Formatter.Register<NotebookMonitor>((monitor, writer) =>
+            {
+                WriteSummary(monitor, writer);
+                WriteChart(monitor, writer);
+                WriteTable(monitor, writer);
+            }, "text/html");
+
+            if (Kernel.Root?.FindKernel("csharp") is { } csKernel)
+            {
+                await LoadExtensionApiAsync(csKernel);
+            }
+        }
+
+        private static async Task LoadExtensionApiAsync(Kernel cSharpKernel)
+        {
+            await cSharpKernel.SendAsync(new SubmitCode($@"#r ""{typeof(AutoMLMonitorKernelExtension).Assembly.Location}""
+using {typeof(NotebookMonitor).Namespace};"));
+        }
+
+        private static void WriteSummary(NotebookMonitor monitor, TextWriter writer)
+        {
+
+            var summary = new List<IHtmlContent>();
+
+            if (monitor.BestTrial != null)
+            {
+                summary.Add(h3("Best Run"));
+                summary.Add(p($"Trial: {monitor.BestTrial.TrialSettings.TrialId}"));
+                summary.Add(p($"Trainer: {monitor.BestTrial.TrialSettings.Pipeline}".Replace("Unknown=>", "")));
+            }
+            if (monitor.ActiveTrial != null)
+            {
+
+                var activeRunParam = JsonSerializer.Serialize(monitor.ActiveTrial.Parameter, new JsonSerializerOptions() { WriteIndented = true, });
+
+                summary.Add(h3("Active Run"));
+                summary.Add(p($"Trial: {monitor.ActiveTrial.TrialId}"));
+                summary.Add(p($"Trainer: {monitor.ActiveTrial.Pipeline}".Replace("Unknown=>", "")));
```

Comment:
@LittleLittleCloud better way to get rid of this "Unknown=>"?

Reply:
Why we want to get rid of  `Unknown`. Maybe we can replace it with `Estimator` for better understanding?

Reply:
It just didn't mean anything to me :) ... does it mean something to you?